packages/components/src/input-control/stories/index.story.tsx

```diff
@@ -9,6 +9,13 @@ import type { Meta, StoryFn } from '@storybook/react';
 import InputControl from '..';
 import { InputControlPrefixWrapper } from '../input-prefix-wrapper';
 import { InputControlSuffixWrapper } from '../input-suffix-wrapper';
+/**
+ * WordPress dependencies
+ */
+import { seen, unseen } from '@wordpress/icons';
+import { useReducer } from '@wordpress/element';
+import Tooltip from '../../tooltip';
+import Button from '../../button';
```

Comment:
This is just our formatting convention (which isn't fully covered by linting), but the import block order is External → WordPress → Internal, and component imports from within the components package go under Internal.